projects/smarti18n/src/lib/services/smarti18n.service.ts

```diff
@@ -103,9 +142,9 @@ export class Smarti18nService {
 		const requests = [config.defaultLocale, config.locale]
 			.filter(locale => !!locale)
 			.map(locale => this.loader.load(locale));
-		const request = requests.length > 1
-			? forkJoin(requests).pipe(map(([defaultLocale, locale]) => ObjectUtils.deepMerge(defaultLocale, locale)))
-			: requests[0];
+		const request = requests.length > 1 ?
+			forkJoin(requests).pipe(map(([defaultLocale, locale]) => ObjectUtils.deepMerge(defaultLocale, locale))) :
+			requests[0];

```

Comment:
usually I prefer to have the operators at the start of the line, because visually looks more elegant AND it's easier to read at a glance what's going on.

Reply:
I left them at the end of the line due to the same reasons haha (imho). Just a matter of preference. Anyway, changed that!